.github/workflows/ci-windows.yml

```diff
@@ -183,12 +183,3 @@ jobs:
             /c/test_aie_vs_cpu \
             $PWD/iree-install \
             $PWD/llvm-aie -v
-
-      - name: Printing IR from aie2xclbin
```

Comment:
I don't have context on this, but why are we removing it?

Reply:
because we don't need it? we have plenty of tests now that test various aspects of every piece that was being tested in that script (xclbinutil/bootgen/cdo/etc).

Reply:
Ok, but doesn't have anything to do with this PR right?

Reply:
It does because I have to update the tests to support both backends. So why update this one if it's useless?